airflow/models.py

```diff
@@ -613,6 +614,384 @@ def __init__(self, dag):
         self.pickle = dag
 
 
+from airflow.ti_dependencies.task_instance_dependency import TIDep
+class EndDateAfterExecutionDateDep(TIDep):
```

Comment:
If these classes aren't models, maybe they should live in `airflow.ti_dependencies.task_instance_dependency`?